src/messageHandlers/helpMessage.js

```diff
@@ -20,5 +20,5 @@ module.exports = function help(robot) {
     HelpService.respondWithVersion,
   );
 
-  robot.respond(/how much .*point.*/i, HelpService.tellHowMuchPointsAreWorth);
+  robot.respond(/how much .*qrafty point.*/i, HelpService.tellHowMuchPointsAreWorth);
```

Comment:
Qrafty isn't a ubiquitous name for hubot and using it directly wouldn't be the best of ideas

Reply:
Thanks, I think I addressed your comment. If not, please share guidance as this is not a familiar code base to me 🙇